db/seeds.rb

```diff
@@ -13,3 +13,8 @@
 User.create(username: "aaron", name: "Aaron", password: "pass")
 User.create(username: "torie", name: "Torie", password: "pass")
 User.create(username: "cole", name: "Cole", password: "pass")
+
+Order.create(user_id: 1, status: "Completed", total: 1000, created_at: "2015-10-18 21:56:18", updated_at: "2015-10-18 21:56:18")
+Order.create(user_id: 2, status: "Paid", total: 10000, created_at: "2015-09-18 21:56:18", updated_at: "2015-09-18 21:56:18")
```

Comment:
Line is too long. [124/80]<br>Separate every 3 digits in the integer portion of a number with underscores(_).